src/javascripts/helpers/data/souvenirData.js

```diff
@@ -0,0 +1,18 @@
+import 'firebase/auth';
+import axios from 'axios';
+import firebaseConfig from '../apiKeys';
+
+const dbUrl = firebaseConfig.databaseURL;
+// GET SOUVENIRS
+const getSouvenirs = (eventId) => new Promise((resolve, reject) => {
```

Comment:
For consistency, lets call the taco `id` since that seems to be the way the other calls are using the event_id